test/saithrift/Makefile

```diff
@@ -32,7 +32,12 @@ CTYPESGEN = /usr/local/bin/ctypesgen
 else
 CTYPESGEN = /usr/local/bin/ctypesgen.py
 endif
-LIBS = -lthrift -lpthread -lsai
+LIBS = -lthrift -lpthread
+ifeq ($(platform),vs)
+LIBS += -lsaivs -lsaimeta -lsaimetadata
```

Comment:
this sounds wrong, why SAI would be dependent on saivs, saimeta and saimetadata ? those libs are generated ins sairedis repo

Reply:
this is for compile saiserver, saithrift.

Reply:
yes, but it still seems wired to me that those libs are here as dependencies, because sairedis is directly dependent on SAI repo, seems to me like this saithrift should be separate repo with dependency on SAI and sairedis and not be located in SAI repo itself

Reply:
sai repo also host ptf test which requires saiserver,